BLUESPAWN-client/src/user/BLUESPAWN.cpp

```diff
@@ -6,8 +6,105 @@
 #include "common/StringUtils.h"
 #include "util/eventlogs/EventLogs.h"
 
+#include "hunt/hunts/HuntT1004.h"
+#include "hunt/hunts/HuntT1037.h"
+#include "hunt/hunts/HuntT1050.h"
+#include "hunt/hunts/HuntT1055.h"
+#include "hunt/hunts/HuntT1060.h"
+#include "hunt/hunts/HuntT1100.h"
+#include "hunt/hunts/HuntT1101.h"
+#include "hunt/hunts/HuntT1103.h"
+#include "hunt/hunts/HuntT1131.h"
+#include "hunt/hunts/HuntT1138.h"
+#include "hunt/hunts/HuntT1182.h"
+#include "hunt/hunts/HuntT1183.h"
+
+#include "monitor/ETW_Wrapper.h"
+
+#include "mitigation/mitigations/MitigateM1042-LLMNR.h"
+#include "mitigation/mitigations/MitigateM1042-WSH.h"
+#include "mitigation/mitigations/MitigateV1093.h"
+#include "mitigation/mitigations/MitigateV1153.h"
+#include "mitigation/mitigations/MitigateV3338.h"
+#include "mitigation/mitigations/MitigateV63597.h"
+#include "mitigation/mitigations/MitigateV63817.h"
+#include "mitigation/mitigations/MitigateV63825.h"
+#include "mitigation/mitigations/MitigateV63829.h"
+#include "mitigation/mitigations/MitigateV72753.h"
+#include "mitigation/mitigations/MitigateV73519.h"
+
 #include <iostream>
 
+IOBase& Bluespawn::io = CLI();
+HuntRegister Bluespawn::huntRecord{ io };
+MitigationRegister Bluespawn::mitigationRecord{ io };
+
+Bluespawn::Bluespawn() {
+	using namespace Hunts;
+
+	HuntT1004* t1004 = new HuntT1004(huntRecord);
+	HuntT1037* t1037 = new HuntT1037(huntRecord);
+	HuntT1050* t1050 = new HuntT1050(huntRecord);
+  HuntT1055* t1055 = new HuntT1055(huntRecord);
+	HuntT1060* t1060 = new HuntT1060(huntRecord);
+	HuntT1100* t1100 = new HuntT1100(huntRecord);
+	HuntT1101* t1101 = new HuntT1101(huntRecord);
+	HuntT1103* t1103 = new HuntT1103(huntRecord);
+	HuntT1131* t1131 = new HuntT1131(huntRecord);
+	HuntT1138* t1138 = new HuntT1138(huntRecord);
+	HuntT1182* t1182 = new HuntT1182(huntRecord);
+	HuntT1183* t1183 = new HuntT1183(huntRecord);
+
+	using namespace Mitigations;
+   
+  MitigateM1042LLMNR* m1042llmnr = new MitigateM1042LLMNR(mitigationRecord);
+	MitigateM1042WSH* m1042wsh = new MitigateM1042WSH(mitigationRecord);
+	MitigateV1093* v1093 = new MitigateV1093(mitigationRecord);
+	MitigateV1153* v1153 = new MitigateV1153(mitigationRecord);
+	MitigateV3338* v3338 = new MitigateV3338(mitigationRecord);
+	MitigateV63597* v63597 = new MitigateV63597(mitigationRecord);
+  MitigateV63817* v63817 = new MitigateV63817(mitigationRecord);
+	MitigateV63825* v63825 = new MitigateV63825(mitigationRecord);
+	MitigateV63829* v63829 = new MitigateV63829(mitigationRecord);
+	MitigateV72753* v72753 = new MitigateV72753(mitigationRecord);
+	MitigateV73519* v73519 = new MitigateV73519(mitigationRecord);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'v73519' is assigned a value that is never used.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=4960363)